client/src/Component/MainTable.js

```diff
@@ -34,86 +40,146 @@ const MainTable = (props) => {
+			<table className="table align-middle table-hover text-nowrap border">
 				<caption className="visually-hidden">Clients</caption>
-				<thead className="table-secondary">
+				<thead className="table-secondary border rounded-top">
 					<tr>
-						<th scope="col">#</th>
+						<th scope="col rounded-start">#</th>
 						<th scope="col">Id</th>
 						<th scope="col">Added</th>
 						<th scope="col">Name</th>
 						<th scope="col">Phone</th>
 						<th scope="col">Bikes needed</th>
 						<th scope="col">Booking status</th>
 						<th scope="col">Pick up date</th>
-						<th scope="col">Residency status</th>
-						<th scope="col">Country of origin</th>
-						<th scope="col">Time in Scotland</th>
-						<th scope="col">Language spoken</th>
-						<th scope="col">English speaker</th>
-						<th scope="col">English skill level</th>
-						<th scope="col">Gender</th>
-						<th scope="col">DOB</th>
-						<th scope="col">Postcode</th>
-						<th scope="col">Referring agency</th>
 					</tr>
 				</thead>
 				<tbody>
-					{props.clientData.map((client) => {
+					{props.clientData.map((client, i) => {
 						return (
-							<tr key={client.client_id}>
-								<td>
-									<Checkbox
-										isChecked={props.selectedClients.includes(client.client_id)}
-										label={client.name}
-										onChange={(checked) => props.onHandleSelectedUserState(checked, client.client_id)}
-									/>
-								</td>
-								<td>{client.client_id}</td>
-								<td>{new Date(client.date_added).toLocaleDateString()}</td>
-								<td className="fw-bold">{client.name}</td>
-								<td>{client.phone_number}</td>
-								<td className="align-bikes-needed">{client.bikes_needed}</td>
-								<td className="fs-5"><span className={bookingStatusStyle[client.booking_status]}>{bookingStatus[client.booking_status]}</span></td>
-								<td>{validDate(client.pick_up_date)}</td>
-								<td>{residencyStatus[client.residency_status]}</td>
-								<td>{client.country_of_origin}</td>
-								<td>{client.time_in_scotland}</td>
-								<td>{client.language_spoken}</td>
-								<td>{speakEnglish(client.english_speaker)}</td>
-								<td>{client.english_skill_level}</td>
-								<td>{client.gender}</td>
-								<td>{validDate(client.date_of_birth)}</td>
-								<td>{client.postcode}</td>
-								<td>{client.referring_agency}</td>
-							</tr>
+							<Fragment key={client.client_id}>
+								<tr>
+									<td className="">
+										<Checkbox
+											isChecked={props.selectedClients.includes(
+												client.client_id
+											)}
+											label={client.name}
+											onChange={(checked) =>
+												props.onHandleSelectedUserState(
+													checked,
+													client.client_id
+												)
+											}
+										/>
+									</td>
+									<td>{client.client_id}</td>
+									<td>{new Date(client.date_added).toLocaleDateString()}</td>
+									<td className="fw-bold d-flex justify-content-start align-items-center">
```

Comment:
```suggestion
									<td className="fw-bold">
```
Flexbox seems to change the size of the cell which is why we're using `fs-4` below. Instead of using flexbox, we could remove it and make the children inline instead

Reply:
Thanks Coung.